added: Firebase, login methods, signup method...

SoftwareDev108 opened the pull request.

kristynamason commented on lib/main.dart:
```diff
@@ -17,8 +17,8 @@ class InvestMe extends StatelessWidget {
         SignUpPage.id: (context) => SignUpPage(),
-        ConfirmationPage.id: (context) => ConfirmationPage(),
-        IntroTutorialPage.id: (context) => IntroTutorialPage(),
+        //ConfirmationPage.id: (context) => ConfirmationPage(),
```
Please do not comment this line out. See Discord or the appropriate comments to handle this.

kristynamason commented on lib/main.dart:
```diff
@@ -17,8 +17,8 @@ class InvestMe extends StatelessWidget {
-        ConfirmationPage.id: (context) => ConfirmationPage(),
-        IntroTutorialPage.id: (context) => IntroTutorialPage(),
+        //ConfirmationPage.id: (context) => ConfirmationPage(),
+        //IntroTutorialPage.id: (context) => IntroTutorialPage(),
```
Please do not comment this line out. See Discord or the appropriate comments to handle this.

kristynamason commented on android/build.gradle:
```diff
@@ -8,6 +8,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.2.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.google.gms:google-services:4.2.0'
```
Are we using Kotlin or Groovy here?

SoftwareDev108 replied:
Aside from the classpath 'com.google.gms:google-services:4.2.0', the classpath dependencies are automatically generated by Android Studio.

kristynamason commented:
Please add a description to this pull request.

kristynamason commented:
Please feel free to make the tickets that correspond with the appropriate Scrum tasks that are completed in this pull request.

SoftwareDev108 commented:
Changes:
- added login methods
- added email field on signup page
- added register button on signup page
- integrated Google Firebase, specifically for the firebase_auth package
- added Google Firebase JSON file to project

kristynamason commented:
Incomplete, but will pull and revise in another pull request.

kristynamason commented:
Incomplete, but will pull and revise in another pull request.
